examples/lib/stories/system/resize_example.dart

```diff
@@ -13,7 +13,7 @@ class ResizingRectangle extends RectangleComponent {
   void onGameResize(Vector2 size) {
     super.onGameResize(size);
 
-    this.size = size * 0.4;
+    this.size = size * .4;
```

Comment:
```suggestion
    this.size = size * 0.4;
```

Reply:
this is copied as is from main - you can fix on a separate PR!